lib/ramble/ramble/application.py

```diff
@@ -106,6 +107,9 @@ def __init__(self, file_path):
         self.license_file = ''
         self.license_inc_name = 'license.inc'
 
+        self.logger = llnl.util.tty.log.log_output(echo=False, debug=tty.debug_level())
+        self.experiment_log_file = None
```

Comment:
I don't think we need this attribute at all here (I'll suggest the other changes below though, that make this not necessary)